wpilibc/athena/src/IterativeRobot.cpp

```diff
@@ -172,7 +172,6 @@ void IterativeRobot::DisabledPeriodic() {
     printf("Default %s() method... Overload me!\n", __FUNCTION__);
     firstRun = false;
   }
-  delayTicks(1);
```

Comment:
you're just removing this?? Yea it looks hacky but I'm guessing it was put there to fix something...